.changeset/hungry-ravens-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/app': minor
+---
+
+Added shopify app function info command to display function run arguments.
```

Comment:
Nit: Since this is user facing, we could frame it as the improvement to the experience. 


```suggestion
Added a `function info` command to display function information.
```